src/test/python/vrs.proto

```diff
@@ -0,0 +1 @@
+../../vrs/schema/vrs.proto
```

Comment:
File not needed when building with maven.

Reply:
Will remove. 👍